src/VisualStudio/Setup.Next/VisualStudioSetup.Next.csproj

```diff
@@ -104,6 +104,9 @@
   <ItemGroup>
     <NuGetPackageToIncludeInVsix Include="StreamJsonRpc" />
     <NuGetPackageToIncludeInVsix Include="Newtonsoft.Json" />
+    <NuGetPackageToIncludeInVsix Include="Microsoft.VisualStudio.Threading" />
+    <NuGetPackageToIncludeInVsix Include="Microsoft.VisualStudio.Validation" />
```

Comment:
ya, limitation of service hub. it will be loaded to service hub process not devenv. and currently service hub can't discover dlls discoverable by devenv process.

there is open bug tracked by service hub team to make that possible (for vsix). until then, we need to have local copy of it for service hub.

before RTM we will figure out what approach we go. if service hub team can make it happen before RC or RTM, we will use that mechanism, if they can't, we probably will create one that installed to Common7/ServiceHubs which can remove this extra dlls, but still put them in vsix for out of band roslyn vsix installation.